python/ray/train/backend.py

```diff
@@ -231,11 +231,12 @@ def _create_placement_group(self):
                 logger.debug("Placement group has started.")
             else:
                 raise TimeoutError(
-                    "Placement group creation timed out. Make sure "
-                    "your cluster either has enough resources or use "
-                    "an autoscaling cluster. Current resources "
-                    "available: {}, resources requested by the "
-                    "placement group: {}".format(
+                    "Placement group creation timed out. Make sure your "
+                    "cluster either has enough resources or use an "
+                    "autoscaling cluster. If you are running on a cluster, "
+                    "make sure you specify an address in `ray.init()`, for example, "
+                    '`ray.init("auto")`. Current resources available: {}, resources '
+                    "requested by the placement group: {}".format(
```

Comment:
Actually sorry, one more thing. Can you also add a sentence on how to increase the timeout by setting the `TRAIN_PLACEMENT_GROUP_TIMEOUT_S` environment variables.